lib/sudo_rails/styling.rb

```diff
@@ -4,6 +4,13 @@ def get_layout
       SudoRails.layout || 'sudo_rails/application'
     end
 
+    def get_render
+      return SudoRails.render if SudoRails.render.is_a?(String)
+      return SudoRails.render.call if SudoRails.render.is_a?(Proc)
```

Comment:
I think we should probably add some comments here in order clarify why this code exists here

Reply:
I've added a comment. Do check if that's enough 😄